### ISSUE TITLE
Edit and save 'Service' description in different language, other than English fails in IE 

### ISSUE THREAD

**Issue by saralavanip** [Digipalvelutehdas/dipor-dashboard#181]
Steps:

1.Visit https://dashboard.digipalvelutehdas.fi

2.Login

3.Create Organization

4.Go to Organization view and select the organization added above

5.Create Department

6.Go to department view

7.Add 'Service'  

  -  Enter Name  
  -  Enter description (in any language eg. English / Suomi)
  -  Select 'status' from top down menu
  -  Enter version 
  -  Click on 'Create'  

8.Go to department view and select the service added above       
9.Click on 'Edit'     
10.Select whole text in description field and delete    
11.Paste a text in different language other than English    

11.Click on 'Save'  

12.Go to department view  

13.Select the 'Service' added above  

Findings:  

- Newly added description in a different language other than English is not saved in Internet Explorer.
- Whereas in other browsers: Google Chrome, Mozilla Firefox and Microsoft Edge, it saves the description in different languages including English. 

Operating systems: 
1. Linux : Ubuntu 14.04.03 LTS (Trusty)
2. Windows 10  

Web Browsers:
1. Internet explorer 11
2. Mozilla Firefox 44.0
3. Google Chrome 48.0.2564.103
4. Microsoft Edge 25.10586.0.0

**Comment by brylie** [comment]
@saralavanip would you please provide an example of the text you used for this test?

**Comment by saralavanip** [comment]
@brylie , please find the example text added in the test description.

**Comment by brylie** [comment]
Are there any unicode characters in the text? E.g. ä, ö, etc?

**Comment by brylie** [comment]
@elnzv can you please test this issue from IE on your local environment?

**Comment by brylie** [comment]
@saralavanip from what source are you copying the text? What is the character encoding?

**Comment by saralavanip** [comment]
Yes, there were unicode characters in the sample texts I used for testing. I also tested with texts in Chinese language ( from google translator) , French, Suomi sample texts from source: https://lipsum.com
